test/suite.bats

```diff
@@ -62,3 +62,17 @@ fixtures suite
   [ "${lines[5]}" = "begin 3 quasi-truth" ]
   [ "${lines[6]}" = "not ok 3 quasi-truth" ]
 }
+
+@test "recursive support (short option)" {
+  run bats -r "${FIXTURE_ROOT}/recursive"
+  [ $status -eq 0 ]
+  echo "$output"
```

Comment:
Let's delete the debugging `echo` lines and check the order of `lines[1]` and `lines[2]` like I mentioned earlier.

Reply:
I didn't actually see a note about checking line ordering, but that is doable.

Reply:
I think it got lost in our back-and-forth, but it's here: https://github.com/bats-core/bats-core/pull/109#discussion_r197925873